src/Sarif.UnitTests/TestData/PrereleaseCompatibilityTransformer/ExpectedOutputs/ComprehensiveFileProperties.sarif

```diff
@@ -69,13 +69,13 @@
         }
       ],
       "resources": {
-        "rules": {
-          "C1": {
+        "rules": [
+          {
```

Comment:
>{ [](start = 10, length = 1)

And now the `rule` object doesn't have an `id` property. In v1 the key was the id, unless there was a collision, so we should have `"id": "C1"` here.

Reply:
hm, good catch

---
In reply to: [249136902](https://github.com/Microsoft/sarif-sdk/pull/1223#discussion_r249136902) [](ancestors = 249136902)